tell newcomers to add `wasm32-unknown-unknown`

<!-- Please only include one item in each issue! -->

This is about:
- [ ] A typo 
- [x] Innaccurate/misleading documentation (e.g. technically incorrect advice)
- [ ] Undocumented code
- [ ] Outdated documentation
- [ ] Other

## **Problem**
<!-- PLEASE INCLUDE A WAY TO LOCATE WHERE THE ISSUE IS, e.g.
 - A Github link to the text
 - The filename and line number
 - An excerpt from the docs that someone could use text search to locate
-->
<!-- What's wrong? -->

I had trivials when running examples of this repository

Just got installed rust, and did what example README.md tells me todo

but faced error messages

<img width="850" alt="image" src="https://github.com/yewstack/yew/assets/46418312/663a873d-e2f7-4779-b983-253131d2a4da">

**I was so scared** but I tried to find out why and found the solution, add ***wasm32-unknown-unknown***.

Thanks to stackoverflow as I always do 

[stackoverflow : error[E0463]: can't find crate for `core` while building rust project for wasm32-unknown-unknown](https://stackoverflow.com/questions/67898431/errore0463-cant-find-crate-for-core-while-building-rust-project-for-wasm32)

## **Details about the solution you'd like** _(Optional)_

add doc to install wasm32 dependencies 

```bash
# thanks to stackoverflow
rustup target add wasm32-unknown-unknown
cargo build --target wasm32-unknown-unknown
```

## **Additional context** _(Optional)_

Most of newcomers could not guess ***wasm32-unknown-unknown*** is something that should or could be installed

literally it says **unknown**

I searched error messages I saw, suspecting it was all about Windows

some  questions and issues of error messages (including blaming Windows )

were quite simliar from what I saw during this process

So it will be very helpful for others and as well as maintainers

## **Questionaire** _(Optional)_
- [ ] I'd like to write this documentation
- [x] I'd like to write this documentation but I'm not sure what's needed
- [ ] I don't have time to add this right now, but maybe later

--- a/examples/README.md
+++ b/examples/README.md
@@ -3,13 +3,8 @@
 ## How to Run
 
 The examples are built with [trunk](https://github.com/thedodd/trunk).
-You can install it with the following command:
-
-```bash
-cargo install --locked trunk
-```
-
-Running an example is as easy as running a single command:
+Once you have the development envrionment fully set up (see [documentation](https://yew.rs/docs/next/getting-started/introduction)),
+running an example is as easy as running a single command:
 
 ```bash
 # move into the directory of the example you want to run

